test/NuGet.Core.FuncTests/Dotnet.Integration.Test/DotnetSourcesTests.cs

```diff
@@ -366,6 +503,76 @@ public void Sources_NoWarnWhenDisableHttpSource()
             }
         }
 
+        [Theory]
+        [InlineData("http://source.test", "true")]
+        [InlineData("http://source.test", "TRUE")]
+        [InlineData("http://source.test", "FALSE")]
+        [InlineData("http://source.test", "invalidString")]
+        [InlineData("http://source.test", "")]
+        [InlineData("https://source.test", "TRUE")]
+        [InlineData("https://source.test", "false")]
+        public void Sources_DisableHttpSourceWithAllowInsecureConnections_NoWarns(string source, string allowInsecureConnections)
```

Comment:
For this test, I wonder if there's value in testing anything other than [InlineData("http://source.test", "false")]. 

@zivkan, @nkolev92 -  thoughts?

Reply:
I agree. 

A lot of this is just testing the parsing of the allowinsecureconnections .  All of that should be unit testable. 

Also https://github.com/NuGet/Client.Engineering/issues/133. 

All of these tests are really really slow. 
I'm guessing this PR adds over a minute of test time altogether. 

I don't see why we need to test every combination.

Reply:
Thanks for reviewing! Makes sense. I removed several tests.